app/containers/ProblemsPage/messages.js

```diff
@@ -16,14 +16,14 @@ export default defineMessages({
   },
   submited: {
     id: 'app.containers.ProblemsPage.submited',
-    defaultMessage: 'Answers submited: {number}',
+    defaultMessage: 'Answers submited: {number, number}',
```

Comment:
może lepiej `{count, number}` bo wymaga to zastanowienia się dlaczego jest 2 razy to samo?